### DISCUSSION TITLE
redirect same page after login

### DISCUSSION THREAD

**Discussion by jamesdesouza** [nextauthjs/next-auth#755] [Help]
**Your question**
I'm using localtest.me domain in development mode.

On _localtest.me:3000/login_ page, I have default **instance** value in order to login. 
After auth process, redirect is working normal!

But if I try to login from _test.localtest.me:3000/login_ page (instance value is "test" now), it redirects me to _localtest.me:3000/login_ page!

**What are you trying to do**
I tought to set redirect url in callbacks like in the codes below
`    callbacks: {
        redirect: async (url, baseUrl) => {
          console.log("CALLBACK redirect", url, baseUrl)
          return Promise.resolve(url)
        },
    },`
Both url and baseUrl are the same! I lost subdomain value.

I can't use env variables as site url because subdomain value will be dynamic.

Thanks,

**Comment by amintabrizi** [comment]
I have same problem!

**Reply by balazsorban44** [reply]
I'm afraid we don't currently support dynamic `NEXTAUTH_URL` values. See https://github.com/nextauthjs/next-auth/issues/600
